crmprtd/__init__.py

```diff
@@ -51,3 +55,26 @@ def f_retry(*args, **kwargs):
             return f(*args, **kwargs)
         return f_retry  # true decorator
     return deco_retry
+
+
+def setup_logging(log_conf, log, error_email, log_level, name):
```

Comment:
Let's change the name of the `log` argument to be `log_filename`, since that's what it actually is.